ThunderCloud/UIAccessibility+ButtonShapes.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  UIAccessibility+ButtonShapes.swift
+//  ThunderCloud
+//
+//  Created by Simon Mitchell on 20/08/2019.
+//  Copyright © 2019 threesidedcube. All rights reserved.
+//
+
+import UIKit
+
+extension UIAccessibility {
+    
+    /// Returns whether button shapes are enabled by the user
+    public static var buttonShapesEnabled: Bool {
```

Comment:
Could be good to get the explanation as to why we're using this method (as iOS doesn't give us a property for this so we have to use this workaround instead?)